src/utils.ts

```diff
@@ -0,0 +1,13 @@
+export const sortByField: any = (arr: any[], field: any) => {
+  return arr.sort((a: any, b: any) => (a[field] > b[field] ? 1 : -1));
+};
+
+export const paginate: any = (
+  arr: any[],
+  pageSize: number,
+  pageNumber: number
+) => {
+  return arr.slice((pageNumber - 1) * pageSize, pageNumber * pageSize);
+};
+
+export const generateId = () => Math.random().toString(36).substr(2, 9);
```

Comment:
Chance to get same value from `Math.random()` low, but never zero. Use it carefully!